src/System.Diagnostics.Process/src/System/Diagnostics/ProcessThread.OSX.cs

```diff
@@ -32,8 +43,11 @@ public TimeSpan PrivilegedProcessorTime
         {
             get
             {
-                // TODO: Implement this
-                throw NotImplemented.ByDesign;
+                Interop.libproc.proc_threadinfo? info = Interop.libproc.GetThreadInfoById((ulong)_threadInfo._threadId);
+                if (info.HasValue)
+                    return new TimeSpan((long)info.Value.pth_system_time);
```

Comment:
Same time question as earlier; are these in the right units for TimeSpan's ctor?